libs/asset-viewer/src/lib/components/asset-viewer-page/asset-viewer-page.component.ts

```diff
@@ -70,6 +69,10 @@ export class AssetViewerPageComponent implements AfterViewInit, OnDestroy {
   private _cd = inject(ChangeDetectorRef);
   private _ngZone = inject(NgZone);
 
+  public isMapInitialised$ = this._store.select(selectIsMapInitialized);
+  public filterLoadingState$ = this._store.select(selectFilterLoadingState);
+  public resultsLoadingState$ = this._store.select(selectSearchLoadingState);
+  public detailLoadingState$ = this._store.select(selectAssetDetailLoadingState);
```

Comment:
Combine these with `combineLatest` and `pipe(map(...))` into something like `isLoading$`. This way, we only need one instance variable for them + It cleans up the template.